products/BellHybrid/services/db/ServiceDB.cpp

```diff
@@ -56,6 +56,35 @@ db::Interface *ServiceDB::getInterface(db::Interface::Name interface)
     return nullptr;
 }
 
+sys::MessagePointer ServiceDB::DataReceivedHandler(sys::DataMessage *msgl, sys::ResponseMessage *resp)
+{
+    auto responseMsg = std::static_pointer_cast<sys::ResponseMessage>(ServiceDBCommon::DataReceivedHandler(msgl, resp));
```

Comment:
I'd split it to method invocation and then cast of the value, as this line has become a bit long
```suggestion
    const auto message = ServiceDBCommon::DataReceivedHandler(msgl, resp);
    auto responseMsg = std::static_pointer_cast<sys::ResponseMessage>(message);
```

Reply:
done